controllers/fenceagentsremediation_controller.go

```diff
@@ -173,6 +174,19 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		r.Log.Info("Finalizer was removed", "CR Name", req.Name)
 		return emptyResult, nil
 	}
+
+	// Validate whetehr the fence agent is supported
+	r.Log.Info("Check if the agent is supported")
+	if isSupported := r.isAgentSupported(far.Spec.Agent); !isSupported {
+		unsuppportedErr := fmt.Errorf("fence agent name didn't match one of the available supported agents")
```

Comment:
typo: s/unsuppported/unsupported

Also, might be clearer using a simpler error message like "unsupported fence agent", even with the name that failed

Reply:
>even with the name that failed

I already add it in the [next log print](https://github.com/medik8s/fence-agents-remediation/pull/116/files#diff-05147ee3f0d225b5a60754b8b9bff2c569deb0a9d7eaddae33af680291d493a1R185) as ` "Fence Agent", far.Spec.Agent`